### PR TITLE
Add correct k_values and relax the knee soft buffer

### PR DESCRIPTION
**Pull request by Ishadijcks**

### CODE REVIEW

File: march_description/urdf/march-iv.xacro [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
     <xacro:property name="knee_rotation_lower_limit" value="${-5*pi/180}"/>
     <xacro:property name="knee_rotation_upper_limit" value="${115*pi/180}"/>
-    <xacro:property name="knee_soft_buffer" value="${5*pi/180}"/> <!-- rad -->
+    <xacro:property name="knee_soft_buffer" value="${4*pi/180}"/> <!-- rad -->
```
Comment (Ishadijcks) [inline march_description/urdf/march-iv.xacro]:
```suggestion
    <xacro:property name="knee_soft_buffer" value="${4.5*pi/180}"/> <!-- rad -->
```

Comment (Ishadijcks) [reply march_description/urdf/march-iv.xacro]: Discussed with Bente, 0.5 overstretching is desired over changing the gaits

File: march_description/urdf/march-iv.xacro [hunk 2]
```diff
@@ -86,7 +86,7 @@
         <axis xyz="-1 0 0"/>
         
         <limit velocity="${hip_aa_velocity_limit}" effort="${hip_aa_effort_limit}" lower="${hip_aa_rotation_lower_limit}" upper="${hip_aa_rotation_upper_limit}"/>
-        <safety_controller soft_lower_limit="${hip_aa_rotation_lower_limit + hip_aa_soft_buffer}" soft_upper_limit="${hip_aa_rotation_upper_limit - hip_aa_soft_buffer}" k_position="100" k_velocity="10" />
+        <safety_controller soft_lower_limit="${hip_aa_rotation_lower_limit + hip_aa_soft_buffer}" soft_upper_limit="${hip_aa_rotation_upper_limit - hip_aa_soft_buffer}" k_position="5" k_velocity="5" />
```
[the remark this answers is not in the document]
Comment (JitskedeVries) [reply march_description/urdf/march-iv.xacro]: I would like to look at the logfiles to see if the desired position could be followed by the exoskeleton, before we decide that these are the values

File: march_description/urdf/march-iv.xacro (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ishadijcks) [reply march_description/urdf/march-iv.xacro]: Disagree, ask Bente

File: march_description/urdf/march-iv.xacro (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ishadijcks) [reply march_description/urdf/march-iv.xacro]: Alright you can do that.

[the remark this answers is not in the document]
Comment (JitskedeVries) [reply march_description/urdf/march-iv.xacro]: It does not look weird in the log files. I think the k values are okay for now. I will come up with better k values with a reasoning soon.

File: march_description/urdf/march-iv.xacro [hunk 3]
```diff
@@ -86,7 +88,7 @@
         <axis xyz="-1 0 0"/>
```
Comment (JitskedeVries) [inline march_description/urdf/march-iv.xacro]:
```suggestion
<!-- TODO: @JitskedeVries  Improve k values-->
```

**Pull request merged**